src/ar.toit

```diff
@@ -35,6 +39,12 @@ DETERMINISTIC-OWNER-ID_  ::= 0
 DETERMINISTIC-GROUP-ID_  ::= 0
 DETERMINISTIC-MODE_      ::= 0b110_100_100  // Octal 644.
 
+/**
+Whether the given $bytes are in the 'ar' format.
+*/
+is-ar bytes/ByteArray -> bool:
```

Comment:
Not sure I like `ar.is-ar` so much. Also, you're not really sure it is in the 'ar' format by just checking the header? Maybe `ar.has-valid-header` or something would convey the limitations better?

Reply:
done.